common/app/views/support/GetClasses.scala

```diff
@@ -39,11 +39,13 @@ object GetClasses {
     )
   }
 
-  def forSubLink(sublink: Sublink): String = RenderClasses(Seq(
-    Some("fc-sublink"),
-    Some(TrailCssClasses.toneClassFromStyle(sublink.cardStyle) + "--sublink"),
-    sublinkMediaTypeClass(sublink)
-  ).flatten: _*)
+  def forSubLink(sublink: Sublink): String = RenderClasses(Map(
+    ("fc-sublink", true),
+    (TrailCssClasses.toneClassFromStyle(sublink.cardStyle) + "--sublink", true),
```

Comment:
enable only if `PillarCards.isSwitchedOff`?

Reply:
For the `tone`, yep good point, I'll stick that check in.